docs/README.md

```diff
@@ -52,30 +52,21 @@ Documentation links:
 
 ## Solution overview
 
-The diagram below illustrates the components of the Kubernetes collection solution.
-
-![solution](/images/k8s_collection_diagram.png)
-
-- **K8S API Server**. Exposes API server metrics.
-- **Scheduler.** Makes Scheduler metrics available on an HTTP metrics port.
-- **Controller Manager.** Makes Controller Manager metrics available on an HTTP metrics port.
-- **node-exporter.** The `node_exporter` add-on exposes node metrics, including CPU,
-  memory, disk, and network utilization.
-- **kube-state-metrics.** Listens to the Kubernetes API server; generates metrics
-  about the state of the deployments, nodes, and pods in the cluster; and exports
-  the metrics as plaintext on an HTTP endpoint listen port.
-- **Prometheus deployment.** Scrapes the metrics exposed by the `node-exporter`
-  add-on for Kubernetes and the `kube-state-metrics` component; writes metrics
-  to a port on the Fluentd deployment.
-- **Fluentd deployment.** Forwards logs and metrics to HTTP sources on a hosted collector.
-  Includes multiple Fluentd plugins that parse and format the metrics and enrich them with metadata.
-- **Events Fluentd deployment.** Forwards events to an HTTP source on a hosted collector.
+The diagrams below illustrate the components of the Kubernetes collection solution.
+
+### Log Collection
+
+![logs](/images/logs.png)
```

Comment:
The Kubernetes node's icon and the Kubernetes API server's icon should have a title, like "k8s node" or "node" and "api-server" or similar.

Reply:
Are you sure it's necessary for the api server? The icon even includes the word "api".